pyproject.toml

```diff
@@ -30,10 +30,10 @@ omegaconf = "^2.1.1"
 # provided by pyg.org, while torch-points-kernels whell is provided by promaton.
 # Please see OSS README for more details on how to build the wheels.
 
-torch-points-kernels = { version = "==0.7.1+anynp", source = "promaton" }
-torch-cluster = {url = "https://data.pyg.org/whl/torch-1.11.0%2Bcu113/torch_cluster-1.6.0-cp38-cp38-linux_x86_64.whl"}
-torch-scatter = {url = "https://data.pyg.org/whl/torch-1.11.0%2Bcu113/torch_scatter-2.0.9-cp38-cp38-linux_x86_64.whl"}
-torch-sparse = {url = "https://data.pyg.org/whl/torch-1.11.0%2Bcu113/torch_sparse-0.6.13-cp38-cp38-linux_x86_64.whl"}
+torch-points-kernels = { url= "https://pypi.dev.promaton.ai/packages/torch_points_kernels-0.7.1-cp310-cp310-linux_x86_64.whl" }
```

Comment:
Just to be sure :thinking: 
Is this one you built + pushed yourself guessing from local from this fork?
https://github.com/promaton/torch-points-kernels

Reply:
Yes, indeed!

Reply:
Actually, `torch-points-kernels` is built and published in a docker container - as explained [here](https://github.com/promaton/ios/tree/main/projects/oss#torch-points-kernels). So, I juggled with the docker stuff to build a version compatible with Python 3.10.